public/js/report.js

```diff
@@ -532,8 +532,8 @@ var HitsTable = React.createClass({
                                         </a>
                                     </td>
                                     {hasName && <td className="text-left">{hit.sciname}</td>}
-                                    <td className="text-right">{hit.qcovs}</td>
-                                    <td className="text-right">{hit.score}</td>
+				    {hasName && <td className="text-right">{hit.qcovs}</td>}
```

Comment:
Need to use a different flag here. `hasName` is true/false depending on whether taxonomy information is available about the hits or not. Since this information is derived from TSV, `hasName` will always be false for imported XML. But `hasName` being false does not necessarily mean that the report was derived from imported XML.